teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] You need to define the `parent_permission` correctly in the `#super` method.

  ```
  super(age, name, parent_permission: parent_permission)
  ```